src/Components/Browser.JS/src/tsconfig.json

```diff
@@ -1,15 +1,6 @@
 {
-  "compilerOptions": {
-    "noImplicitAny": false,
-    "noEmitOnError": true,
-    "removeComments": false,
-    "sourceMap": true,
-    "target": "es5",
-    "lib": ["es2015", "dom"],
-    "strict": true
-  },
-  "exclude": [
-    "node_modules",
-    "wwwroot"
-  ]
+    "extends": "../tsconfig.base.json",
+    "include": [
+        "./**/*"
```

Comment:
Could you clarify why this is here? The compiler normally know what's included based on imports.